src/importexport/musicxml/internal/musicxml/importmxmlpass2.cpp

```diff
@@ -3653,7 +3657,8 @@ void MusicXMLInferredFingering::addToNotes(std::vector<Note*>& notes) const
     for (size_t i = 0; i < m_fingerings.size(); ++i) {
         // Fingerings in reverse order
         addTextToNote(-1, -1,
-                      m_fingerings[m_fingerings.size() - 1 - i], m_placement, u"", -1, u"", u"", TextStyleType::FINGERING,
+                      m_fingerings[m_fingerings.size() - 1 - i], m_placement, u"", -1, u"", u"",
+                      notes[i]->color(), TextStyleType::FINGERING,
```

Comment:
Not sure, might as well just use black here

Reply:
I think I would prefer just black indeed

Reply:
Me too, to be honest ;-)
Done